src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs

```diff
@@ -113,6 +113,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             string source = _categoryName ?? stateSourceValue;
             if (exception is FunctionIndexingException && _eventManager != null)
             {
+                // publish failure metric here?
```

Comment:
Given this log and [this accumulation of indexing failures](https://github.com/Azure/azure-functions-host/blob/2c830a03dede60feb7aa074dd524d3cb02e7834c/src/WebJobs.Script/Host/ScriptHost.cs#L897), I don't think this is a complete failure/host shutdown, and I'm not sure how we'd want to count this in the metrics. Thoughts?